packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -611,32 +322,30 @@ export abstract class QueryEditorStore {
         },
       },
     );
+    this.queryCreatorState = new QueryCreatorState(this, undefined);
   }
 
-  get isViewProjectActionDisabled(): boolean {
-    return false;
-  }
+  abstract get label(): string;
```

Comment:
if it's not always useful/appropriate to show this, should we make it optional? e.g. the default implementation always return undefined, a plugin/extension can override if needs be